test/users/test_profile_picture.py

```diff
@@ -0,0 +1,27 @@
+from PIL import Image
+from flaskapp import models
+from test.main.base_classes import BaseUser
+from test.main.utils import test_post_request
+
+
+class UserAccountTestCase(BaseUser):
+    def test_set_profile(self):
+        self.login()
+        im1 = Image.open("profile1.png")
```

Comment:
Here you have to specify absolute path 
add this line in __init__.py of test folder
```python
TEST_PATH = os.path.dirname(os.path.abspath(__file__))
```
now add following line in test_profile_picture
```python
from test import TEST_PATH
file = os.path.join(TEST_PATH, "users", "profile1.png")
```
and use it to open